### ISSUE TITLE
maqetta can't be used in win7 64bit locally.

### ISSUE THREAD

**Issue by yanzenggao** [maqetta/maqetta#3538]
In win7 64bit local use, can't activate a user. How to fix that?

**Comment by billreed63** [comment]
I am running on windows 7 64bit and have no problems, please provide step by step instructions on recreating your issue.

**Comment by JonFerraiolo** [comment]
Yes, please provide step-by-step instructions. In particular, which ZIP download are you using? 

If you are using http://maqetta.org/downloads/maqetta-release-7.0.1.zip, then to start off you click "Register" and then enter any email address and password to create a new account. With this version, there is no email activation - just type in a new user name and off you go.

If you are using http://maqetta.org/downloads/maqetta-hosted-release-7.0.1.zip, then activation requires integration with some sort of sendmail capability. I don't know the details about how to get that to work.

**Comment by jhpedemonte** [comment]
@yanzenggao, have you got this working yet?

**Comment by yanzenggao** [comment]
Yes. I used hosted first. And when I switched to released version I forgot to delete the cookies of chrome.------------------ 原始邮件 ------------------发件人: "Javier Pedemonte"notifications@github.com;发送时间: 2012年10月23日(星期二) 晚上8:04收件人: "maqetta/maqetta"maqetta@noreply.github.com; 抄送: "yanzenggao"376384683@qq.com; 主题: Re: [maqetta] maqetta can't be used in win7 64bit locally. (#3538)@yanzenggao, have you got this working yet?

```
          —
          Reply to this email directly or view it on GitHub.
```
